src/main/java/com/microsoft/lst_bench/input/Task.java

```diff
@@ -43,15 +44,7 @@ public interface Task {
   @Nullable String getCustomTaskExecutor();
 
   @JsonProperty("custom_task_executor_arguments")
-  @Nullable CustomTaskExecutorArguments getCustomTaskExecutorArguments();
-
-  @Value.Immutable
-  @JsonSerialize(as = ImmutableCustomTaskExecutorArguments.class)
-  @JsonDeserialize(as = ImmutableCustomTaskExecutorArguments.class)
-  interface CustomTaskExecutorArguments {
-    @JsonProperty("dependent_task_batch_size")
```

Comment:
Since this includes removal of `dependent_task_batch_size`, the `workload.json` schema file be updated as well?

Reply:
done!